### PR TITLE
feat: expose createTheme as createCartoTheme

### PR DESCRIPTION
**Pull request by AdriSolid**
Expose `cartoTheme` as `createCartoTheme`.

Related with `documentation`: https://github.com/CartoDB/documentation/pull/78

### CODE REVIEW

File: packages/react-ui/src/theme/carto-theme.js [hunk 1]
```diff
@@ -1175,16 +1175,10 @@ export const cartoThemeOptions = {
   }
 };
 
-export function createTheme(options = {}) {
```
Comment (VictorVelarde) [inline packages/react-ui/src/theme/carto-theme.js:1178]: Are we sure we can remove default options?

**Comment by VictorVelarde** [comment]
Please add a note to changelog and link here the PR with the doc change

File: packages/react-ui/src/theme/carto-theme.js (hunk 1 above)
Comment (AdriSolid) [reply packages/react-ui/src/theme/carto-theme.js:1178]: As described in `documentation` PR, the way to overwrite `cartoThemeOptions` is equal for both implementations in this way.

Also, passing to `createTheme` and object like:
```javascript
const mainColor = { palette: { primary: { main: '#80000' }}}
```
results in an error. Do you have any other idea to handle this?

Comment (VictorVelarde) [reply packages/react-ui/src/theme/carto-theme.js:1178]: What if we leave the optional param, and just expose the method?

That way we could also allow overriding certain properties, so we would eventually get 3 options for the user:

- call to createTheme without params (one line from the client)
- call to createTheme with certain options (maybe just colors)
- completely custom case, using just cartoThemeOptions (what is currently being used from template)

/cc @borja-munoz

**Comment by VictorVelarde** [comment]
We should talk more about this one with @borja-munoz, before exposing it

**Comment by VictorVelarde** [comment]
Not a clear advantage on the change, so I'm closing this

**Pull request closed without merging**